src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs

```diff
@@ -339,26 +331,12 @@ private static class CultureInfoMapper
         {
             ///  Dictionary of CultureInfo.DisplayName, CultureInfo.Name for cultures that have changed DisplayName over releases.
             ///  This is to workaround an issue with CultureInfoConverter that serializes DisplayName (fixing it would introduce breaking changes).
-            private static volatile System.Collections.Generic.Dictionary<string, string> s_cultureInfoNameMap;
-
-            public static string GetCultureInfoName(string cultureInfoDisplayName)
-            {
-                if (s_cultureInfoNameMap == null)
-                {
-                    InitializeCultureInfoMap();
-                }
-
-                if (s_cultureInfoNameMap.ContainsKey(cultureInfoDisplayName))
-                {
-                    return s_cultureInfoNameMap[cultureInfoDisplayName];
-                }
+            private static readonly Dictionary<string, string> s_cultureInfoNameMap = CreateMap();
```

Comment:
With this change, volatile isn't necessary anymore... it's almost the opposite of readonly, where readonly effectively says "this will never change so feel free to cache it as long as you like" and volatile kind of says "this can change at any moment and it's important you use the absolute latest".